src/data/content/design-system/fondamenti/versionamento.yaml

```diff
@@ -228,6 +228,15 @@ components:
       components:
         - name: TextImageCta
           text: |
+            -------
+            ### v1.0.0-alpha.11
```

Comment:
🚩 note to myself @Fupete: remember to update this to the correct version number before merging 🤔

Reply:
Or better to remove the changes to this version... and do a dedicated PR.